supplementary_style_guide/style_guidelines/release-notes.adoc

```diff
@@ -63,108 +63,453 @@ Introduce each release note with a heading that summarizes the release note. Thi
 * Avoid definitions in headings unless necessary for clarity. For example, use definitions to disambiguate different meanings of the same name: "The `journald` system role can tune the performance of the `journald` service".
 * Do not start the heading with a gerund. Use gerunds only for procedural content.
 
-== Release note categories
+[[jira-links-release-notes]]
+== Linking to Jira in release notes
```

Comment:
Not all jira references are links - we discussed an unlinked jira number to be an acceptable reference to a private ticket.

```suggestion
== References to Jira in release notes
```